src/integration/telegram/telegram_bot.py

```
@@ -0,0 +1,349 @@
+            **Intelligence Confidence:** `{intelligence_summary.get('learning_confidence', 0):.2f}`
+            """
+            
+            await message.reply_text(status_message, parse_mode='Markdown')
+            
+        except Exception as e:
+            logger.error(f"Error in status command: {e}")
+            try:
+                message = update.message or update.effective_message
+                if message:
+                    await message.reply_text("Error retrieving status. Please try again.")
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)